### PR TITLE
feat: 1:1 call participants view - Part 1: create layout (AR-1370)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1370" title="AR-1370" target="_blank">StoryAR-1370</a>  1:1 call participants view
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Description

This is the first PR of this feature and it aims to create the layout needed to implement it.

For now, we are only displaying informations of the current user, next PR will be for displaying other user infos.

One other change also made is by making the bottom sheet collapse and expand on tap anywhere in the screen

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

Make a call with some one

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

Wire 2022-06-09 at 4_25 PM

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-651/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-651/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-651/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3582293)(3582293)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3582293)(3582293)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt [hunk 1]
```diff
@@ -73,8 +91,7 @@ private fun OngoingCallContent(
                 ) { }
             },
             sheetShape = RoundedCornerShape(topStart = dimensions().corner16x, topEnd = dimensions().corner16x),
-            backgroundColor = MaterialTheme.wireColorScheme.ongoingCallBackground,
-            sheetPeekHeight = dimensions().defaultSheetPeekHeight,
+            sheetPeekHeight = 0.dp,
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt:94]: `Peek`? 🧐

**Review by gongracr — APPROVED** [review]
Just added a couple of tiny comments. Otherwise looking grrreat 🐯

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-651/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3588767)(3588767)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3588767)(3588767)-dev-release.apk)

**Pull request merged**